dataset_loaders/data_augmentation.py

```diff
@@ -410,6 +413,18 @@ def random_transform(x, y=None,
     crop_size: tuple
         The size of crop to be applied to images and masks (after any
         other transformation).
+    crop_mode: string
+        The crop strategy, could be either 'random' or 'smart'.
+        The 'random' mode randomly places the crop in the image.
+        The 'smart' mode centers the crop in one of the locations where
+        the non-background masks are more present through time (in case
+        of sequences) or in the image otherwise. To do so it looks for a
+        label called 'background' or 'void' to retrieve the mask id or
+        assumes the id of the background mask to be 0.
```

Comment:
The code is not looking for a background label!!

Fix that and change the description as follows:
[...]
the non-background masks are present more often (in the image, or over time in the case of sequences). To do so it looks into `self.mask_labels` for a label called 'background' or 'void' to retrieve the mask id or assumes the id of the background mask to be 0.